### ISSUE TITLE
Webtorrents sometimes stops downloading

### ISSUE THREAD

**Issue by vankasteelj** [butterproject/butter-desktop#525]
I found no references about it on webtorrent.

1) Open an issue on webtorrent
2) If not a bug, then we need to try to detect speed = 0 and download isn't done, then start downloading again. It shouldnt be hard

**Comment by xaiki** [comment]
yeah, i've experienced that bug a few time using webtorrent cmdline client... we'd need to see if destruction of the webtorrent object is not needed for this to work properly =/
